src/App.tsx

```diff
@@ -1,26 +1,49 @@
+/* eslint-disable no-console */
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from './app/store';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { todosActions } from './features/todos';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const dispatch = useDispatch<AppDispatch>();
```

Comment:
Create a custom hook to get dispatch with the corresponding type
https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-dispatch-type